src/schema/v2/artist/index.ts

```diff
@@ -309,6 +310,56 @@ export const ArtistType = new GraphQLObjectType<any, ResolverContext>({
           )
         },
       },
+      badges: {
+        type: new GraphQLNonNull(
+          new GraphQLList(
+            new GraphQLNonNull(
+              new GraphQLObjectType({
+                name: "ArtistBadge",
+                fields: {
+                  label: {
+                    type: new GraphQLNonNull(GraphQLString),
+                  },
+                  type: {
+                    type: new GraphQLEnumType({
+                      name: "ArtistBadgeType",
+                      values: {
+                        ACTIVE_SECONDARY_MARKET: {
+                          value: "ACTIVE_SECONDARY_MARKET",
+                        },
+                        HIGH_AUCTION_RECORD: {
+                          value: "HIGH_AUCTION_RECORD",
+                        },
+                      },
+                    }),
+                  },
+                  entities: {
+                    type: new GraphQLNonNull(
+                      new GraphQLList(new GraphQLNonNull(GraphQLString))
+                    ),
+                  },
+                },
+              })
+            )
+          )
+        ),
```

Comment:
Just a note to mention that this general schema form matches what we do for `artist#insights`

Reply:
forgot to mention, thanks